### PR TITLE
Local LDAP server

### PR DESCRIPTION
**Pull request by buchi**
Add a local LDAP server that is run with Docker Compose.
This way we no longer depend on connectivity to the 4teamwork LDAP.

## Checklist

_Everything has to be done/checked. Checked but not present means the author deemed it unnecessary._

- [ ] Changelog entry
- [ ] Link to issue (Jira or GitHub) and backlink in issue (Jira)

### CODE REVIEW

**Review by lukasgraf — APPROVED** [review]
LGTM! 👍 Just some small comments.
Thanks for the extensive updates to the README!

File: docker-compose.yml [hunk 1]
```diff
@@ -94,12 +94,14 @@ services:
...
       - OGDS_DSN=postgresql://${OGDS_DB_USER}:${OGDS_DB_PASSWORD}@${OGDS_DB_HOST}/${OGDS_DB_NAME}
-      - LDAP_PROFILE=DS389
+      - LDAP_PROFILE=OpenLDAP
       - LDAP_URL=${LDAP_SERVER_URI}
       - LDAP_BIND_DN=${LDAP_BIND_DN}
       - LDAP_BIND_PASSWORD=${LDAP_BIND_PASSWORD}
       - LDAP_USER_BASE_DN=${LDAP_USER_BASE_DN}
       - LDAP_GROUP_BASE_DN=${LDAP_GROUP_BASE_DN}
+      - 'USER_ATTRIBUTE_MAPPING={"external_id":"entryUUID","username":"uid","firstname":"givenName","lastname":"sn","email":"mail","display_name":"displayName","organization":"o","department":"ou"}'
+      - 'GROUP_ATTRIBUTE_MAPPING={"external_id":"entryUUID","groupname":"cn","title":"description"}'
```
[the remark this answers is not in the document]
Comment (buchi) [reply docker-compose.yml]: Removed mapping

File: README.rst [hunk 2]
```diff
@@ -3,169 +3,62 @@ opengever.core
...
-For CentOS, the ``tetex-latex`` package contains the ``pdflatex`` binary. For
-local development on OS X we recommend the `MacTeX distribution <http://www.tug.org/mactex/>`_.
-
-There is a 4teamwork internal `devdocs LaTeX section <https://devdocs.4teamwork.ch/latex/>`_
-on how to install ``pdflatex`` with our own fonts.
-
-HAProxy
-^^^^^^^
+    $ cp .env.sample .env
+    $ docker compose up -d
```
[the remark this answers is not in the document]
Comment (buchi) [reply README.rst:28]: This is not necessary. The database is created automatically on startup of the ogds container.

**Comment by buchi** [comment]
I've now added default env values in the compose file. A `.env` file is no longer needed, but can be used to customize settings.

Removed employee user names from example content.

**Pull request merged**